core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1340,6 +1340,15 @@ public boolean argumentMustBeScalar(int ordinal) {
   @Deprecated // to be removed before 2.0
   public static final SqlFunction JSON_STORAGE_SIZE = SqlLibraryOperators.JSON_STORAGE_SIZE;
 
+  @Deprecated // to be removed before 2.0
+  public static final SqlFunction JSON_INSERT = SqlLibraryOperators.JSON_INSERT;
+
+  @Deprecated // to be removed before 2.0
+  public static final SqlFunction JSON_REPLACE = SqlLibraryOperators.JSON_REPLACE;
```

Comment:
Revert the change.

Reply:
> Revert the change.

Currently, it cannot be temporarily deleted. I will modify the entire JSON function later.

Reply:
???

Reply:
> 

Because I found that deleting these lines will cause the `testJsonInsert`, `testJsonReplace`, and `testJsonSet` in `RelToSqlConverterTest` to fail even if `sql(query).withMysql ().Ok (expected)`.

Reply:
> ???

Because I found that deleting these lines will cause the `testJsonInsert`, `testJsonReplace`, and `testJsonSet` in `RelToSqlConverterTest` to fail even if `sql(query).withMysql ().Ok (expected)`.

Reply:
Then fix that.